session-windows/flinksql/README.md

```diff
@@ -0,0 +1,205 @@
+
+  ```shell
+  docker exec -it flink-sql-client sql-client.sh
+  ```
+
+  Finally, run following SQL statements to create the `clicks` table backed by Kafka running in Docker, populate them with
+  test data, and then run a statement displaying the session windows.
+
+  ```sql
+  CREATE TABLE clicks (
+            ip STRING,
+            url STRING,
+            click_ts TIMESTAMP(3),
+            WATERMARK FOR click_ts AS click_ts
+  ) WITH (
+      'connector' = 'kafka',
+      'topic' = 'clicks',
+      'connector' = 'kafka',
+      'properties.bootstrap.servers' = 'broker:9092',
+      'scan.startup.mode' = 'earliest-offset',
+      'key.format' = 'raw',
+      'key.fields' = 'ip',
+      'value.format' = 'json',
+      'value.fields-include' = 'EXCEPT_KEY'
+  );
+ ``` 
+```sql
+ INSERT INTO clicks VALUES
+        ('9.62.201.241','/acme/jeep-stuff/', TO_TIMESTAMP('2023-07-09 01:00:00')),
+        ('122.65.213.141', '/farm-for-all/chickens/', TO_TIMESTAMP('2023-07-09 02:00:10')),
+        ('122.65.213.141', '/farm-for-all/chickens/', TO_TIMESTAMP('2023-07-09 02:00:20')),
+        ('122.65.213.141', '/farm-for-all/chickens/', TO_TIMESTAMP('2023-07-09 02:01:00')),
+        ('9.62.201.241', '/acme/jeep-stuff/', TO_TIMESTAMP('2023-07-09 01:00:30')),
+        ('9.62.201.241', '/acme/jeep-stuff/', TO_TIMESTAMP('2023-07-09 01:01:00')),
+        ('21.229.87.11', '/amc-rio/movies/', TO_TIMESTAMP('2023-07-09 09:00:00')),
+        ('234.112.107.50', '/trips/packages/', TO_TIMESTAMP('2023-07-09 12:00:00')),
+        ('21.229.87.11', '/amc-rio/movies/', TO_TIMESTAMP('2023-07-09 09:00:30')),
+        ('122.65.213.141', '/farm-for-all/tractors/', TO_TIMESTAMP('2023-07-09 02:30:00')),
+        ('122.65.213.141', '/farm-for-all/tractors/', TO_TIMESTAMP('2023-07-10 02:31:00'));
+```
+```sql
+SELECT url,
+    COUNT(url) AS visited_count,
+    window_start,
+    window_end
+FROM TABLE(SESSION(TABLE clicks PARTITION BY url, DESCRIPTOR(click_ts), INTERVAL '2' MINUTES))
+GROUP BY url, window_start, window_end;
+```
+
+  The query output should look like this:
+
+  ```plaintext
+              url              visited_count                           window_start                window_end      
+/acme/jeep-stuff/                       3                  2023-07-09 01:00:00.000   2023-07-09 01:03:00.000
+/farm-for-all/chickens/                 3                  2023-07-09 02:00:10.000   2023-07-09 02:03:00.000
+/farm-for-all/tractors/                 1                  2023-07-09 02:30:00.000   2023-07-09 02:32:00.000
+/amc-rio/movies/                        2                  2023-07-09 09:00:00.000   2023-07-09 09:02:30.000
+/trips/packages/                        1                  2023-07-09 12:00:00.000   2023-07-09 12:02:00.000
+  ```
+
+  When you are finished, clean up the containers used for this tutorial by running:
+
+  ```shell
+  docker compose -f ./docker/docker-compose-flinksql.yml down
+  ```
+
+</details>
+
+<details>
```

Comment:
thinking we should remove this section from the PR so that the tutorial is valid on merge. and then either lve it as is, or open a second WIP PR that has this change in it and waits on CC support